docs/91_version_history.rst

```diff
@@ -34,30 +34,60 @@ Release Date: 2023-12-28
 
     - New Features:
 
-
+       - A new REST API has been added to the public API to allow performance counters to be fetched and reset. This API is intended for diagnostic purposes only.
+       - Improvements have been made to the way load issues on the storage service are logged. 
+       - Documentation has been added that describes how to extend the API to include custom URLs for executing control functions. This documentation also shows how these are then called using the graphical user interface.
 
     - Bug Fix:
 
+       - An issue with the PostgreSQL storage plugin when very large numbers of readings are ingested, more than 4294967296, has now been resolved.
+       - An issue with services shutting down rather than restarting when they fail to get a valid bearer token has been resolved.
+       - The user interface for creating write API endpoints was incorrectly requiring both a constant and a variable when only one is required, this is now resolved.
+       - A problem that meant parameters to set point control operations were not correctly sent to south plugins written in Python has been resolved.
 
 
 - **GUI**
 
     - New Features:
 
+       - The user interface has been upgraded to use Angular version 16.
+       - The configuration section of the user interface that allows for instance wide configuration has been improved with a single tree navigation item and improved visual feedback.
+       - A link to the documentation has been added to the Control API pages of the user interface.
 
 
     - Bug Fix:
 
+       - An issue that could cause some datapoint to display incorrectly in the user interface graph when multiple assets are displayed and those assets have data points with the same name in both assets has been resolved.
+       - An issue in the user interface that meant exporting data as a CSV file created incorrect files if any of the data point names contained a comma has been fixed.
+       - A broken link to the documentation in the control pipeline user interface page of the user interface has been fixed.
 
 
 - **Plugins**
 
     - New Features:
 
+       - The benchmark south plugin has been enhanced to increase the load that can be placed during testing.
+       - The S2 OPCUA south plugin has been enhanced to allow filtering of nodes using regular expressions on the name of the nodes.
```

Comment:
It's "...on the Browse Name of the nodes." This is OPC UA terminology which is important in this case.